arch/x86/soc/intel_quark/quark_se/Kconfig.defconfig.series

```diff
@@ -17,13 +17,15 @@ config TOOLCHAIN_VARIANT
 	default "" if !X86_IAMCU
 
 config PHYS_RAM_ADDR
-	default 0xA8006400
+	default 0xA8007000 if X86_MMU
```

Comment:
The 80K RAM on the SOC is shared between the ARC and x86 sides. So if you change where the X86 RAM starts you need to make a corresponding change on the ARC side configuration.